release.nix

```diff
@@ -14,7 +17,8 @@ in
 
 with (import (fixedNixpkgs + "/pkgs/top-level/release-lib.nix") {
   inherit supportedSystems scrubJobs nixpkgsArgs;
-  packageSet = import ./.;
+  packageSet = args:
+    removeAttrs (import ./. args) skipPackages;
```

Comment:
line 91 is responsible for adding the docker images, so this wont actually remove them

Reply:
Fixed